internal/service/advancedclustertpf/resource_test.go

```diff
@@ -1,28 +1,56 @@
 package advancedclustertpf_test
 
 import (
+	"fmt"
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
 
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/service/advancedclustertpf"
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
 )
 
+const (
+	resourceName = "mongodbatlas_advanced_cluster.test"
+)
+
+func ChangeReponseNumber(responseNumber int) resource.TestCheckFunc {
+	changer := func(*terraform.State) error {
+		advancedclustertpf.SetCurrentClusterResponse(responseNumber)
+		return nil
+	}
+	return changer
+}
+
 func TestAccAdvancedCluster_basic(t *testing.T) {
 	resource.ParallelTest(t, resource.TestCase{
 		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
 		Steps: []resource.TestStep{
 			{
-				Config: configBasic(),
+				Config: configBasic(""),
+			},
+			{
+				Config: configBasic("accept_data_risks_and_force_replica_set_reconfig = \"2006-01-02T15:04:05Z\""),
+				Check:  resource.ComposeTestCheckFunc(ChangeReponseNumber(2)),
+			},
+			{
+				Config:                               configBasic("accept_data_risks_and_force_replica_set_reconfig = \"2006-01-02T15:04:05Z\""),
```

Comment:
why do we add this attribute here, is it needed for import?

Reply:
You are right, not needed